internal/snykclient/monitordeps_test.go

```diff
@@ -0,0 +1,45 @@
+package snykclient_test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/snyk/cli-extension-sbom/internal/mocks"
+	"github.com/snyk/cli-extension-sbom/internal/snykclient"
+)
+
+var exampleScanResult = snykclient.ScanResult{
+	Name:   "Bob",
+	Policy: "",
+	Facts: []snykclient.ScanResultFact{
+		{Type: "depgraph", Data: ""},
```

Comment:
nit: not to diverge too much from reality, can we make this
```suggestion
		{Type: "depGraph", Data: struct{}{}},
```
`"data"` isn't actually a string, but a depgraph object...